lib/ae_mdw_web/controllers/aex141_controller.ex

```diff
@@ -0,0 +1,45 @@
+defmodule AeMdwWeb.Aex141Controller do
+  @moduledoc """
+  Controller for specific AEX141 endpoints.
+  """
+
+  use AeMdwWeb, :controller
+
+  alias AeMdw.Aex141
+  alias AeMdw.Error.Input, as: ErrInput
+  alias AeMdw.Validate
+  alias AeMdwWeb.FallbackController
+  alias AeMdwWeb.Plugs.PaginatedPlug
+  alias AeMdwWeb.Util, as: WebUtil
+
+  alias Plug.Conn
+
+  import AeMdwWeb.Helpers.AexnHelper
+
+  plug(PaginatedPlug)
+  action_fallback(FallbackController)
+
+  @spec nft_owner(Conn.t(), map()) :: Conn.t()
+  def nft_owner(conn, %{"contract_id" => contract_id, "token_id" => token_id}) do
+    with {:ok, contract_pk} <- Validate.id(contract_id, [:contract_pubkey]),
+         {token_id, ""} <- Integer.parse(token_id),
```

Comment:
This is going to return `:error` which is not handled by the `FallbackController`.

I'd perhaps send the contract and token ID directly to `Aex141.fetch_nft_owner/2` function and let it return an error if any of these things are invalid

Reply:
Done here in the controller because `call_contract` also returns `:error`

Reply:
This way endpoint params validation can be done also closer to border (entry door)